test/e2e/network/service.go

```diff
@@ -43,6 +43,36 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+const (
+	defaultServeHostnameServicePort = 80
+	defaultServeHostnameServiceName = "svc-hostname"
+)
+
+var (
+	defaultServeHostnameService = v1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: defaultServeHostnameServiceName,
+		},
+		Spec: v1.ServiceSpec{
+			Ports: []v1.ServicePort{{
+				Port:       int32(defaultServeHostnameServicePort),
+				TargetPort: intstr.FromInt(9376),
+				Protocol:   "TCP",
+			}},
+			Selector: map[string]string{
+				"name": defaultServeHostnameServiceName,
+			},
+		},
+	}
+)
+
+func getServeHostnameService(name string) *v1.Service {
```

Comment:
This is something we should put into util. But I guess it is fine for now :)

Reply:
This makes perfect sense. Actually not only this function but also default settings and constants should go to service_util too. But that will cause some change. I will clean it up after this PR get merged.

Reply:
SGTM